theano/gpuarray/tests/test_subtensor.py

```diff
@@ -321,7 +321,7 @@ def test_tensor_float16(self):
 
 class TestGpuAllocDiag(test_basic.TestAllocDiag):
     def __init__(self, name):
-        return test_basic.TestAllocDiag.__init__(
+        test_basic.TestAllocDiag.__init__(
```

Comment:
I guess this doesn't return anything?

Reply:
Conceptually, an `__init__()` method is not allowed to return anything, which technically means it's required to return `None`, which is what happens by default without an explicit `return` value. https://docs.python.org/3/reference/datamodel.html#object.__init__

So this change clarifies the situation and removes a PyCharm inspection warning.